src/util/cmirdmautils.h

```diff
@@ -53,6 +53,8 @@ typedef struct ncpystruct{
 
   int ncpyOpInfoSize;
 
+  unsigned int rootNode; // used only for Broadcast, -1 for p2p operations
```

Comment:
Why unsigned if -1 is valid?

Reply:
Ah, yes. I'll remove the unsigned. (-1 is valid and will be used for p2p operations, which is when the field is unused).

Reply:
Done.